package.json

```diff
@@ -74,20 +74,21 @@
     "@revert/color": "*",
     "@start/plugin": "*",
     "@types/node": "^14.0.0",
-    "@types/react-native": "^0.64.0",
+    "@types/react-native": "^0.65.9",
     "@types/ws": "^7.2.6",
     "eslint": "^7.0.0",
     "graphiq": "*",
-    "react": "17.0.1",
-    "react-native": "0.64.0",
-    "react-native-device-info": "^6.0.1",
-    "react-native-svg": "^12.1.0",
+    "react": "17.0.2",
+    "react-native": "0.66.1",
+    "react-native-device-info": "^8.4.1",
+    "react-native-svg": "^12.1.1",
     "tsfn": "*",
     "typescript": "^4.0.0",
     "ws": "^7.3.1"
   },
   "scripts": {
-    "start": "packages/nextools/start-preset/src/cli/index.js"
+    "start": "packages/nextools/start-preset/src/cli/index.js",
+    "postinstall": "perl -pi -e \"s/'React'/'React-Core'/g\" ./node_modules/react-native-svg/RNSVG.podspec ./node_modules/react-native-view-shot/react-native-view-shot.podspec && perl -pi -e \"s|}.*node_modules/react-native/cli.js|}/../../../../../node_modules/react-native/cli.js|g\" ./node_modules/react-native/react.gradle"
```

Comment:
I'm doing two things here:
```
perl -pi -e \"s/'React'/'React-Core'/g\" ./node_modules/react-native-svg/RNSVG.podspec ./node_modules/react-native-view-shot/react-native-view-shot.podspec
```
Here I'm replacing "React" with "React-Core" in podspec of "react-native-svg" and "react-native-view-shot" dependencies. For "react-native-svg" that [change is merged since July](https://github.com/react-native-svg/react-native-svg/blob/develop/RNSVG.podspec#L19), but it was not released yet and the same is for "react-native-view-shot" [merged, but not released](https://github.com/gre/react-native-view-shot/blob/master/react-native-view-shot.podspec#L18) yet :/

```
perl -pi -e \"s|}.*node_modules/react-native/cli.js|}/../../../../../node_modules/react-native/cli.js|g\" ./node_modules/react-native/react.gradle
```
Then I'm adding some "../../" in "react-native/react.gradle". This is related to [this change in React Native](https://github.com/facebook/react-native/commit/502b81904998b800f2d960bb4a8e244988c72958#diff-9f1ec5621e9be76a8225dc77bab4cbb7dc2ec6d79cd6e4b54e0af5e2b8aa6a2aL32).
They stopped executing `def cliPath = ["node", "-e", "console.log(require('react-native/cli').bin);"].execute([], projectDir).text.trim()` and rely only on relative path and we started getting an error, because our Android project folder is not in the root, but also inside node_modules.

Why in perl? Because it's fast, I could fit it in one line and it's preinstalled on every machine compatible with POSIX standard.